composer.json

```diff
@@ -36,7 +36,7 @@
   "require-dev": {
     "brianium/paratest": "^4.0",
     "dms/phpunit-arraysubset-asserts": "^0.1.0",
-    "laravel/lumen-framework": "^5.8|^6.0|^7.0",
+    "laravel/lumen-framework": "^5.7|^6.0|^7.0|^8.0",
```

Comment:
I'm guessing this was a merge error or something.

``suggestion
    "laravel/lumen-framework": "^5.8|^6.0|^7.0|^8.0",
```